docs/herbarium/working-with-herbarium.md

```diff
@@ -0,0 +1,139 @@
+---
+id: working-with-herbarium
+title: Working with Herbarium
+sidebar_label: Working with Herbarium
+slug: /herbarium/working-with-herbarium
+---
+
+### Advanced Options
+
+`options`:
```

Comment:
It is not clear for what it is for. 

It should be clear it is a advanced options of `herbarium.requireAll(options)`